apps/frontend/src/libs/components/table/styles.module.css

```diff
@@ -51,6 +51,11 @@
 	overflow-y: auto;
 }
 
+.table .table-body--no-scroll {
+	padding-right: 10px;
+	overflow-y: hidden;
+}
```

Comment:
I considered adding a line using `::after`, but we can't apply `position: relative` to the table as it would disrupt the popover. I can set the width to 101% to account for the scrollbar space, but this would result in a slight movement, which I think is better to avoid